src/core.js

```diff
@@ -1410,6 +1410,7 @@ define([
         refreshMessages();
 
         $props.show();
+        this.data.windowManager.adjustToWindow();
```

Comment:
Funny, I just fixed this bug a different way in f9767db965b5740a7702224b612cb651057209ad